diff -up httpretty-1.0.2/tests/functional/test_passthrough.py.orig httpretty-1.0.2/tests/functional/test_passthrough.py
--- httpretty-1.0.2/tests/functional/test_passthrough.py.orig	2020-03-10 04:08:59.000000000 +0100
+++ httpretty-1.0.2/tests/functional/test_passthrough.py	2020-03-27 11:40:51.716073339 +0100
@@ -36,6 +36,7 @@ def http():
     return sess
 
 
+@skip
 def test_http_passthrough():
     url = 'http://httpbin.org/status/200'
     response1 = http().get(url)
@@ -58,6 +59,7 @@ def test_http_passthrough():
     (response4.content).should.equal(response1.content)
 
 
+@skip
 def test_https_passthrough():
     url = 'https://raw.githubusercontent.com/gabrielfalcao/httpretty/master/COPYING'
 
